src/datepicker/datepicker-day-view.ts

```diff
@@ -4,16 +4,22 @@ import {NgbDateStruct} from './ngb-date-struct';
 @Component({
   selector: '[ngbDatepickerDayView]',
   styles: [`
-    :host {      
+    :host {
       text-align: center;
-      padding: 0.185rem 0.25rem;      
+      width: 2rem;
+      height: 2rem;
+      line-height: 2rem;      
       border-radius: 0.25rem;
     }
+    :host.outside {
+      opacity: 0.5;
+    }
   `],
   host: {
     '[class.bg-primary]': 'selected',
     '[class.text-white]': 'selected',
     '[class.text-muted]': 'isMuted()',
+    '[class.outside]': 'isMuted()',
```

Comment:
`text-muted` is the bootstrap class. I wanted to add 50% opacity on top of it, so two options:

- `[style.opacity]="isMuted() ? 0.5 : 1"`
- add the css class

I picked the last one, not sure which one is better though